src/html/index.html.template

```diff
@@ -44,7 +44,7 @@
       }
       #ha-launch-screen .ha-launch-screen-spacer-top {
         flex: 1;
-        margin-top: calc( 2 * max(var(--safe-area-inset-bottom, 0px), 48px) + 46px );
+        margin-top: calc( 2 * max(var(--safe-area-inset-top), 48px) + 46px );
```

Comment:
Previous comment on this line: https://github.com/home-assistant/frontend/pull/25566#discussion_r2102962001